SPEC.md

```diff
@@ -98,61 +98,104 @@ A build tool is something which reads one or more input artifacts and writes one
 ## Specifications
 ### Artifact ID
 
-GitBOM uses the git object id (as a blob) for an artifact as its artifact ID.
+A hash function over the []byte of an artifact satisfies the canonical, unique, immutable requirements for an artifact identifier.
```

Comment:
While I understand the go-ism as well, and it's been useful shorthand in slides, for the spec I am inclined to agree with Andrew's suggestion.

Perhaps this will suffice in the generic?

```suggestion
Because two artifacts are equivalent if and only if their binary representations are equal, a hash function may be applied to the binary representation of an artifact to yield an identifier which satisfies the canonical, unique, and immutable requirements of artifact identifiers.
```

Reply:
Unfortunately, github doesn't let me open a comment or suggestion in an area of the document not being modified by this PR ...

So I'll write it here ...

> Two artifacts are equivalent if and only if their binary representations are equal. This can be expressed in pseudocode with the following function: `[]byte(artifact1) == []byte(artifact2)`

Reply:
@alilleybrinker Does the above work for you?